backend/src/main/java/com/brs/backend/configuration/SecurityConfig.java

```diff
@@ -45,8 +45,6 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
     @Bean
     CorsConfigurationSource corsConfigurationSource() {
         CorsConfiguration configuration = new CorsConfiguration();
-        //    TODO : Remove following hardcode once deployed the new config version
-        configuration.setAllowedOrigins(Arrays.asList(frontendUrl, "https://apl.aragorn-media-server.duckdns.org"));
```

Comment:
Should be kept as

‘configuration.setAllowedOrigins(Arrays.asList(frontendUrl));’

Reply:
done